src/frontend/context/SøknadContext.ts

```diff
@@ -148,7 +149,7 @@ const [SøknadProvider, useSøknad] = createUseContext(
                 // Ny behandling er lastet som ikke har fullført søknad-steget.
                 tilbakestillSøknad();
             }
-        }, [åpenBehandling]);
+        }, [åpenBehandling.søknadsgrunnlag]);
```

Comment:
Problemet med dette er at det går i beina på det effekten løser: at når vi får åpenBehandling-objektet fra backend så skal vi sjekke om det følger med et søknadsgrunnlag fra backend og fylle det inn i skjemaet. 

Caset det skal løse er at man står inne på registrer søknad-siden og bytter aktiv åpen behandling - da må vi sjekke om vi skal preutfylle skjemaet eller om vi skal nullstille det (fordi skjemaet inneholder data for en behandling man ikke lenger ser på)

Da er det mer relevant å se på om dette faktisk vil skje - at konteksten mottar en ny åpen behandling - utenom ved første render. Hvis ikke så kan vi fjerne dependency arrayet også vil effekten kun kjøres ved første sidelast. Men hvis det fortsatt funksjonelt er mulig å bytte åpen behandling mens man står på denne siden så må vi se på en annen måte å oppdage at vi bytter behandling og trigge denne effekten. Å lytte på søknadsgrunnlag blir i alle fall ikke riktig

Reply:
Den eneste måten jeg klarer å se at man får åpenBehandling utenom første render er dersom man navigerer rett til denne siden og komponenten rekker å rendre før vi får svaret fra backend... 🤔